testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/AbstractClientPoliciesTest.java

```diff
@@ -889,6 +890,10 @@ protected Object createSecureSigningAlgorithmForSignedJwtEnforceExecutorConfig()
         return new SecureSigningAlgorithmForSignedJwtEnforceExecutor.Configuration();
     }
 
+    protected Object createConfidentialClientAcceptExecutorConfig() {
```

Comment:
As mentioned above, this method can be probably removed (similar methods for other executors as well) and just use ClientPolicyExecutorConfiguration. WDYT?

Reply:
@mposolda I would like to remove such method as I've mentioned just above.

BTW, we already had such executors that do not have any configuration other than this `ConfidentialClientAcceptExecutor`, namely, `SecureRedirectUriEnforceExecutor`, `SecureResponseTypeExecutor`, `SecureSessionEnforceExecutor`, `SecureSigningAlgorithmEnforceExecutor` and  `SecureSigningAlgorithmForSignedJwtEnforceExecutor`.
I would like to create the follow-up JIRA ticket and try to do the same refactoring as this `ConfidentialClientAcceptExecutor`.